js/src/lib/inferenceSnippets/servePython.ts

```diff
@@ -41,6 +42,30 @@ import io
 from PIL import Image
 image = Image.open(io.BytesIO(image_bytes))`;
 
+export const snippetTextToAudio = (model: ModelData): string => {
+	if (model.library_name === "transformers") {
+		return `def query(payload):
+	response = requests.post(API_URL, headers=headers, json=payload)
+	return response.content
+audio_bytes = query({
+	"inputs": ${getModelInputSnippet(model)},
+})
+# You can access the audio with IPython.display for example
+from IPython.display import Audio
```

Comment:
Is this shown in the snippet? Does it fit the box? It seems cut off in your screenshot

Reply:
Here's what I get if I copy the snippet:

```
import requests

API_URL = "https://api-inference.huggingface.co/models/reach-vb/test-mms-tts-model"
headers = {"Authorization": "Bearer hf_reach-vb"}

def query(payload):
	response = requests.post(API_URL, headers=headers, json=payload)
	return response.content
audio_bytes = query({
	"inputs": "The answer to the universe is 42",
})
# You can access the audio with IPython.display for example
from IPython.display import Audio
Audio(audio_bytes)
```

Reply:
So it is a UI issue I guess?